doc/manual/source/man/dnst-key2ds.rst

```diff
@@ -0,0 +1,42 @@
+dnst-key2ds
+===============
+
+Synopsis
+--------
+
+:program:`dnst key2ds` [``OPTIONS``] ``<KEYFILE>``
```

Comment:
Throughout the docs it's at least inconsistent whether its ```[``OPTIONS``]``` or ``` ``[OPTIONS]`` ```.

In my opinion, it should be like in the uutils docs (which makes sense that I like it because I made it):

![image](https://github.com/user-attachments/assets/03883cd5-9cdd-4349-82a8-36eee41a4fcd)

Just one big code block with all possible invocations.

Reply:
> Throughout the docs it's at least inconsistent whether its [``OPTIONS``] or ``[OPTIONS]``.

I'll solve the inconsistencies.

The big code block with possible invocations sound nice, but might not be able to implement that today.